DESCRIPTION

```diff
@@ -29,4 +29,6 @@ Imports:
     dplyr (>= 0.8.1),
     tibble (>= 2.1.3),
     stats,
-    utils
+    utils,
+    S4Vectors,
+    mia
```

Comment:
do we need mia?

Reply:
TreeSummarizedExperiment pkg is still missing from DESCRIPTION; and I am not sure if mia import is needed at all?

Reply:
little mixup there, it's fixed now